docs/framework/net-native/index.md

```diff
@@ -26,27 +26,30 @@ manager: "wpickett"
   
  Typically, apps that target the .NET Framework are compiled to intermediate language (IL). At run time, the just-in-time (JIT) compiler translates the IL to native code. In contrast, [!INCLUDE[net_native](../../../includes/net-native-md.md)] compiles Windows apps directly to native code. For developers, this means:  
   
--   Your apps will provide the superior performance of native code.  
+-   Your apps feature the performance of native code. Usually, performance will be superior to code that is first compiled to IL and then compiled to native code by the JIT compiler. 
   
 -   You can continue to program in C# or Visual Basic.  
   
 -   You can continue to take advantage of the resources provided by the .NET Framework, including its class library, automatic memory management and garbage collection, and exception handling.  
   
  For users of your apps, [!INCLUDE[net_native](../../../includes/net-native-md.md)] offers these advantages:  
   
--   Fast execution times  
+-   Faster execution times for the majority of apps and scenarios
   
--   Consistently speedy startup times  
+-   Faster startup times for the majority of apps and scenarios 
   
 -   Low deployment and update costs  
   
 -   Optimized app memory usage  
-  
- But [!INCLUDE[net_native](../../../includes/net-native-md.md)] involves more than a compilation to native code. It transforms the way that .NET Framework apps are built and executed. In particular:  
+
+> [!IMPORTANT]
+> For the vast majority of apps and scenarios, .NET Native offers significantly faster startup times and superior performance when compared to an app compiled to IL or to an NGEN image. However, your results may vary. To ensure that your app has benefited from the performance enhancements of .NET Native, you should compare its performance with that of the non-.NET Native version of your app.
```

Comment:
can we include any links to where people can find out more about how they can do this comparison?